just_audio_windows/windows/player.hpp

```diff
@@ -57,315 +59,534 @@ auto TO_WIDESTRING = [](std::string string) -> std::wstring {
     return std::wstring();
   }
   int32_t target_length =
-      ::MultiByteToWideChar(CP_UTF8, MB_ERR_INVALID_CHARS, string.data(),
-                            static_cast<int32_t>(string.length()), nullptr, 0);
+    ::MultiByteToWideChar(CP_UTF8, MB_ERR_INVALID_CHARS, string.data(),
+      static_cast<int32_t>(string.length()), nullptr, 0);
   if (target_length == 0) {
     return std::wstring();
   }
   std::wstring utf16_string;
   utf16_string.resize(target_length);
   int32_t converted_length =
-      ::MultiByteToWideChar(CP_UTF8, MB_ERR_INVALID_CHARS, string.data(),
-                            static_cast<int32_t>(string.length()),
-                            utf16_string.data(), target_length);
+    ::MultiByteToWideChar(CP_UTF8, MB_ERR_INVALID_CHARS, string.data(),
+      static_cast<int32_t>(string.length()),
+      utf16_string.data(), target_length);
   if (converted_length == 0) {
     return std::wstring();
   }
   return utf16_string;
 };
 
-class AudioPlayer
-{
-private:
-    /* data */
+
+class BetterEventSink {
```

Comment:
What about we rename this to `JustAudioEventSink`?

Reply:
I copied the name from Android code, but a rename is reasonable.